cns/ipamclient/api.go

```diff
@@ -0,0 +1,73 @@
+package ipamclient
```

Comment:
Why do we even have this file? All these definitions are duplicates from ipam.api, right?